h2o-core/src/main/java/water/rapids/ast/prims/assign/AstAppend.java

```diff
@@ -1,25 +1,27 @@
 package water.rapids.ast.prims.assign;
 
-import water.H2O;
 import water.fvec.Frame;
 import water.fvec.Vec;
 import water.rapids.Env;
 import water.rapids.Val;
-import water.rapids.vals.ValFrame;
 import water.rapids.ast.AstPrimitive;
 import water.rapids.ast.AstRoot;
+import water.rapids.vals.ValFrame;
 
 /**
+ * Attach a named column(s) to a destination frame.
+ *
+ * Syntax: destinationFrame (sourceFrame columnName)+
  */
 public class AstAppend extends AstPrimitive {
   @Override
   public String[] args() {
-    return new String[]{"dst", "src", "colName"};
+    return new String[]{"dst (src colName)+"};
   }
 
   @Override
   public int nargs() {
-    return 1 + 3;
+    return -1;
```

Comment:
Would you also consider leaving number of parameters fixed and allowing the last one to be a list of columns

Reply:
yup, but then you still need a list for column names.